plugin/build_baron.go

```diff
@@ -178,6 +178,29 @@ func BbGetConfig(settings *evergreen.Settings) map[string]evergreen.BuildBaronPr
 }
 
 func BbGetProject(settings *evergreen.Settings, projectId string) (evergreen.BuildBaronProject, bool) {
+	flags, err := evergreen.GetServiceFlags()
+	if err != nil {
+		return evergreen.BuildBaronProject{}, false
+	}
+	if flags.PluginAdminPageDisabled {
+		lastGoodVersion, err := model.FindVersionByLastKnownGoodConfig(projectId, -1)
+		if err != nil || lastGoodVersion == nil {
```

Comment:
if lastGoodVersion == nil, I think we'd want to continue to looking in the project ref instead